.pipelines/cni/load-test-templates/create-cluster-template.yaml

```diff
@@ -15,13 +16,15 @@ steps:
       inlineScript: |
         set -ex
         make -C ./hack/aks azcfg AZCLI=az REGION=${{ parameters.region }}
-
+        

```

Comment:
nit: white space should be removed. You can setup your IDE to remove those for you on each save of the file.